includes/Media.php

```diff
@@ -294,6 +273,60 @@ public static function rest_api_init() {
 		);
 	}
 
+	/**
+	 * Force media attachment as string instead of the default array.
+	 *
+	 * @param array $prepared Prepared data before response.
+	 *
+	 * @return string
+	 */
+	public function get_callback_media_source( $prepared ) {
+		$id = $prepared['id'];
+
+		$terms = wp_get_object_terms( $id, self::STORY_MEDIA_TAXONOMY );
+		if ( is_array( $terms ) && $terms ) {
+			$term = array_shift( $terms );
+
+			return $term->slug;
```

Comment:
Let's optimize this while we're at it:

```suggestion
			return array_shift( $terms )->slug;
```

Reply:
I don't like that, it is not clear.

Reply:
Really we should check the element has a slug before extracting it here.

Reply:
Shouldn't this list only include `WP_Term` objects though?